src/pages/Part/PartVotePage.tsx

```diff
@@ -1,6 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLocation } from 'react-router';
+
+import styled from '@emotion/styled';
+
+import { Headline } from '../../styles/Typography';
+import PartVoteCard from './PartVoteCard';
+import MainBtn from '../../components/MainBtn';
+
+import { DUMMY_MEMBERS } from '../../constants/dummyData';
+
 const PartVotePage = () => {
-  return <div>파트장 투표창 페이지</div>;
+  const navigate = useNavigate();
+  const { state } = useLocation();
+
+  const [selectedMember, setSelectedMember] = useState<{
```

Comment:
selectedMember의 team이름과 user이름은 겹칠 염려가 있고 (물론 현 세오스에서는 없겠지만, 팀과 이름이 동일한 사람이 존재할 수 있음),
만일 한 멤버당 저장하는 정보의 양이 많아진다면 코드가 복잡해질 염려가 있으니 선택된 멤버의 idx 값을 저장해서 비교해보면 어떨까요?

Reply:
이 부분 ui 동작만 할 수 있도록 넣어둔 것이어서 아마 추후 api 양식에 맞추거나 memberId로 비교하지 않을까 싶습니다 ㅎㅎ.

예상)
```suggestion
  const [selectedMemberId, setSelectedMemberId] = useState<string>(0)
```